src/sql/parts/grid/views/editData/editData.component.ts

```diff
@@ -367,11 +367,16 @@ export class EditDataComponent extends GridParentComponent implements OnInit, On
 			columnDefinitions: resultSet.columnInfo.map((c, i) => {
 				let isLinked = c.isXml || c.isJson;
 				let linkType = c.isXml ? 'xml' : 'json';
+
+				// Ensure that the column names are rendered as text and not HTML
+				let columnNameElement = document.createElement('div');
+				columnNameElement.textContent = c.columnName;
+
 				return {
 					id: i.toString(),
 					name: c.columnName === 'Microsoft SQL Server 2005 XML Showplan'
 						? 'XML Showplan'
-						: c.columnName,
+						: columnNameElement.innerHTML,
```

Comment:
It's probably better to just escape the text isn't it?

Reply:
Elsewhere us and vscode are just using vscode strings escape function.

Reply:
ok just updated it to use the escape function